qml/main.qml

```diff
@@ -3,12 +3,6 @@ import QtQuick.Controls 2.3
 import QtQuick.Controls.Material 2.1
 import QtQuick.Dialogs 1.3
 
-import QtCharts 2.3
-
-import QtGraphicalEffects 1.0
-
-import QtQml.Models 2.2
```

Comment:
They are necessary for windows deployment, without it the *deployqt binaries will not provide the necessary files to execute run these libraries.